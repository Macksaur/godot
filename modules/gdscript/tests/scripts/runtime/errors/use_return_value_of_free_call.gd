func test():
	var obj := Node.new()
	print(obj.free())
