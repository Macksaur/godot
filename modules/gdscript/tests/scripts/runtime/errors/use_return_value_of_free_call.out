GDTEST_OK
>> WARNING
>> Line: 3
>> RETURN_VALUE_VOID
>> Cannot get return value of call to "free()" because it returns "void".
<null>
